tests/test_test_utils.py

```diff
@@ -305,3 +305,11 @@ def test_noop(self) -> None:
     assert _hello_world_str == text
 
     await client.close()
+
+
+@pytest.mark.parametrize("hostname", ["127.0.0.1", "localhost", "::1"])
+async def test_test_server_hostnames(hostname, loop) -> None:
+    app = _create_example_app()
+    server = _TestServer(app, host=hostname, loop=loop)
+    async with server:
+        pass
```

Comment:
Maybe add some assertion?
```suggestion
        pass

    assert server.host == hostname
```

Reply:
Sounds good, added an assertion checking TestServer host against expected hostname.

Reply:
FYI it's a good tone to apply suggested changes via GitHub UI since it automatically preserves the authorship by adding `Co-Authored-By:` commit message trailer. This can then be pulled and squashed manually if needed.